specification/web/resource-manager/Microsoft.CertificateRegistration/stable/2024-11-01/AppServiceCertificateOrders.json

```diff
@@ -0,0 +1,1801 @@
+              "readOnly": true
+            },
+            "appServiceCertificateNotRenewableReasons": {
+              "description": "Reasons why App Service Certificate is not renewable at the current moment.",
+              "type": "array",
+              "items": {
+                "enum": [
+                  "RegistrationStatusNotSupportedForRenewal",
+                  "ExpirationNotInRenewalTimeRange",
+                  "SubscriptionNotActive"
+                ],
+                "type": "string",
+                "x-ms-enum": {
+                  "name": "ResourceNotRenewableReason",
+                  "modelAsString": true
+                }
+              },
+              "readOnly": true
+            },
+            "nextAutoRenewalTimeStamp": {
+              "format": "date-time",
+              "description": "Time stamp when the certificate would be auto renewed next",
+              "type": "string",
+              "readOnly": true
+            },
+            "contact": {
+              "$ref": "#/definitions/CertificateOrderContact",
+              "description": "Contact info",
+              "readOnly": true
+            }
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "AppServiceCertificateOrderCollection": {
+      "description": "Collection of certificate orders.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AppServiceCertificateOrder"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "AppServiceCertificateOrderPatchResource": {
+      "description": "ARM resource for a certificate order that is purchased through Azure.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../Microsoft.Web/stable/2024-11-01/CommonDefinitions.json#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "AppServiceCertificateOrderPatchResource resource specific properties",
+          "required": [
+            "productType"
+          ],
+          "type": "object",
+          "properties": {
+            "certificates": {
+              "description": "State of the Key Vault secret.",
+              "type": "object",
+              "additionalProperties": {
+                "$ref": "#/definitions/AppServiceCertificate"
+              }
+            },
+            "distinguishedName": {
+              "description": "Certificate distinguished name.",
+              "type": "string",
+              "x-ms-mutability": [
+                "create",
+                "read"
+              ]
+            },
+            "domainVerificationToken": {
```

Comment:
>domainVerificationToken [](http://example.com/codeflow?start=13&length=23)

If this is a secret, it must have the x-ms-secret extension. Also, PATCH properties are not allowed to be readOnly.